### ISSUE TITLE
Error when selecting row border style from the dropdown in advanced row properties

### ISSUE THREAD

**Issue by gkubisa** [tinymce/tinymce#4124]
**Do you want to request a *feature* or report a *bug*?**

BUG

**What is the current behavior?**

Error thrown and displayed in the dev console and the dropdown does not disappear when selecting a row border style. Clicking outside closes the dropdown and the style is selected.

```
Uncaught TypeError: Cannot read property 'parseStyle' of undefined
    at e (plugin.min.js:3)
    at k.<anonymous> (plugin.min.js:1)
    at c.i [as fire] (tinymce.min.js:8)
    at k.fire (theme.min.js:2)
    at k.fire (theme.min.js:2)
    at k.<anonymous> (theme.min.js:5)
    at c.i [as fire] (tinymce.min.js:8)
    at k.fire (theme.min.js:2)
    at HTMLDivElement.c (theme.min.js:1)
    at m (tinymce.min.js:2)
```

**If the current behavior is a bug, please provide the steps to reproduce and if possible a minimal demo of the problem via [fiddle.tinymce.com](http://fiddle.tinymce.com/) or similar.**

1. Go to https://jsfiddle.net/cz9qxyqr/
2. Create a 2x2 table
3. Put the cursor in the top left cell
4. Go to advanced table row properties
5. Try to select a border style

**What is the expected behavior?**

The border style should be selected, the dropdown should be hidden and no errors should be reported in the console.

**Which versions of TinyMCE, and which browser / OS are affected by this issue? Did this work in previous versions of TinyMCE?**

TinyMCE 4.7.4
Chrome 62
Ubuntu 16.04

I have not tested other versions, browsers nor OSes.

**Comment by joshualouisjenson** [comment]
I am also experiencing this.

TinyMCE 4.7.4
Chrome 63.0.3239.84
Windows 10 - Version 10.0.15063

**Comment by lnewson** [comment]
Hi,

I'm unable to reproduce this issue in TinyMCE 4.9.10 or 5.2.2 using the details provided, however looking back at the changelog it appears this issue was fixed in TinyMCE 4.7.5. As such I'm going to close this, however if you're still experiencing issues then let us know and we'll reopen this issue.

Thanks!
